bbfreeze/recipes.py

```diff
@@ -234,7 +234,9 @@ def recipe_matplotlib(mf):
         mf.copyTree(dp, "matplotlibdata", m)
 
     mf.import_hook("matplotlib.numerix.random_array", m)
-    backend_name = 'backend_' + matplotlib.get_backend().lower()
+    # the import hook expects a string (matplotlib in later versions has
+    # all rcParams, including the backend as unicode)
+    backend_name = 'backend_' + str(matplotlib.get_backend().lower())
```

Comment:
would it be more general to apply the `str()` conversion here https://github.com/schmir/bbfreeze/blob/master/bbfreeze/freezer.py#L306 instead?